.github/workflows/build-and-push.yml

```diff
@@ -22,12 +24,23 @@ jobs:
           BASE_IMAGE_ROW=${row/BASE_IMAGE_NAME = /}
           echo ::set-output name=image_name::$BASE_IMAGE_ROW
 
+      - name: Checkout into generated section
+        run: |
+          git checkout --progress --force -B generated refs/remotes/origin/generated
```

Comment:
Do we really want to use the generated branch? I mean, I don't use it and I'm not sure there is a check that the branch is always up to date.

Reply:
It is a good question. I did not test whether dist-gen works on Ubuntu, yet ;) I can try it and then we can call 'make generate-all' and take the resources correspondingly. But it needs time to investigate it.

Reply:
Fix by the latest commit. We generate sources by each merge.